utils/RunONNXModel.py

```diff
@@ -160,27 +174,37 @@ def check_non_negative(argname, value):
 parser.add_argument(
     "--atol", type=str, default="0.01", help="Absolute tolerance for verification."
 )
+parser.add_argument(
+    "-onnxmlir-args",
+    type=str,
+    default="",
+    help="Options passed to onnxmlir. Used with --use-onnxmlir",
```

Comment:
I did not find the option `--use-onnxmlir`, did I miss it?

Also, did you not want to change the name?